Framework/Lumberjack/CocoaLumberjack.h

```diff
@@ -0,0 +1,39 @@
+//
+//  CocoaLumberjack.h
+//  CocoaLumberjack
+//
+//  Created by Andrew Mackenzie-Ross on 3/02/2015.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+//! Project version number for CocoaLumberjack.
+FOUNDATION_EXPORT double CocoaLumberjackVersionNumber;
+
+//! Project version string for CocoaLumberjack.
+FOUNDATION_EXPORT const unsigned char CocoaLumberjackVersionString[];
```

Comment:
This one actually uses <CocaLumberjack/*> imports instead of “" as its the framework umbrella header (won’t work for static libs). We could rename either this header or the original and then have this import the other header? What do you think?